### ISSUE TITLE
Changelogs in solder

### ISSUE THREAD

**Issue by Tomson124** [TechnicPack/TechnicSolder#280]
After having the possibility to upload images for the modpack, it would be nice if there would be an option to do the modpack changelog handling through solder. Maybe you already thought of it @GenPage but if not I think it would be a good idea and a easement to modpack authors.

Tomson

**Comment by TheDoctorsLife** [comment]
I would like to see this feature.

**Comment by Razorskills** [comment]
I wish github had an "I Agree" button

**Comment by sct** [comment]
This is something I would like to see happen but we will have to have to expand the platform API to allow changelogs to be pushed from wherever.

**Comment by TheDoctorsLife** [comment]
That's true, but you can always make it so the Platform will accept only domains linked to Owner accounts of the modpacks. (AKA I couldn't use my solder install to over ride some else's changelog sort of thing)

**Comment by sct** [comment]
I mean there still needs to be an API regardless and if there is going to be an API to do that its going to be implemented properly.

**Comment by CannibalVox** [comment]
Why not just have the platform web side have a feature to autogen changelogs from solder?  Give it an old build # and it'll diff that to the current release build & populate the editor with some logs that you can then edit if you'd like.

**Comment by Razorskills** [comment]
^ Yes please?

**Comment by Tankernn** [comment]
I would love to see the auto-changelog feature added.

**Comment by JamiePhonic** [comment]
even if it just shows the change log on solder itself so it can be copy/pasted to the technic platform at the users leasure, that would be awesome

**Comment by Isigiel** [comment]
A very basic implementation, but that coul easily be done on solder or platform side (this is JS only)
https://youtu.be/qaeT-jBdQiM

**Comment by skwerlman** [comment]
@Isigiel that vid is private

**Comment by Isigiel** [comment]
@skwerlman I changed the URL, somhow Youtube changed it

**Comment by Isigiel** [comment]
I have finished this by now, [Changelogs](http://isigiel.github.io/changelogger/), go there and just enter any platform api link, so you can try-out yourself

**Comment by Razorskills** [comment]
That is awesome!

**Comment by ET-Bent** [comment]
@Isigiel +1 this is awesome! If this will someday be implemented in either solder or the platform itself, together with the long awaited file upload function (1 year now :D) that would be awesome!
